cs-amazon/src/test/java/io/cloudslang/content/amazon/utils/InputsUtilTest.java

```diff
@@ -55,6 +58,14 @@ public void getMaxInstancesCountBlank() {
     }
 
     @Test
+    public void getStringsList() {
+        assertNull(InputsUtil.getStringsList("",","));
+        assertEquals(2, InputsUtil.getStringsList("2,3",",").size());
+        assertTrue(InputsUtil.getStringsList("2,3",",").contains("2"));
```

Comment:
I think you can call getStringsList only once and use a local variable to do all the checks